source/Halibut.Tests/FailureModesFixture.cs

```diff
@@ -28,21 +28,17 @@ static DelegateServiceFactory GetDelegateServiceFactory()
 
         [Test]
         [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testListening: false)]
-        [LatestClientAndPreviousServiceVersionsTestCases(testNetworkConditions: false, testListening: false)]
```

Comment:
As we were making the latest client only version, we found this test case, as it was going to be an issue.

But looking closer, `LatestClientAndPreviousServiceVersionsTestCases` is still just the latest client (when you have 'no service'). 

So it looked like it was an unrequired double up on effectively the same test case. 

That is why it was removed.

Reply:
lol yep